include/common_table.h

```diff
@@ -38,7 +38,7 @@ typedef struct CommonTable
 } Table;
 
 typedef enum
-{ KEY, SERVER } TableType;
+{ KEY, SERVER, TEST } TableType;
```

Comment:
What's the purpose of the ```TEST``` type? I think the only use of it I noticed was in a test of ```table_destroy```.

Reply:
This is a test type to be used in the test of ```table_destroy``` and later in other table unit test if needed.